jspsych-uil-utils.js

```diff
@@ -191,4 +193,27 @@ var uil = {};
         return has_touchscreen
     }
 
+    /* ********* file loading *********** */
+
+    // The directory this script lives in
+    let scriptDir = document.currentScript.baseURI.split('/').slice(0, -1).join('/');
+
+    /**
+     * Load in a separate sub-library.
+     * @param library The name of the file to load in
+     */
+    context.loadLibrary = function (library) {
```

Comment:
I would like to see context.loadLibrary as a private function, that is not defined inside of the object uil, unless we have reasons that the users of the library will use it, or we like to pass it to other scripts inside the jspsych-uil-libraries. In that case we could opt for a pythonic function context._loadLibrary() {//}

Reply:
I decided to make this a public function (as opposed to all other functions related to loading files) because I could see that this specific function would be handy to have public in the future. 

We might have a sub-library that might not be great to have loaded in every experiment, for whatever reason. Being able to load it in from anywhere isn't a bad API to have.

Reply:
Ok keep it public.